src/Appwrite/GraphQL/Schema.php

```diff
@@ -98,27 +98,36 @@ protected static function api(App $utopia, callable $complexity): array
             foreach ($routes as $route) {
                 /** @var Route $route */
 
-                $namespace = $route->getLabel('sdk.namespace', '');
-                $method = $route->getLabel('sdk.method', '');
-                $name = $namespace . \ucfirst($method);
+                /** @var \Appwrite\SDK\Method $sdk  */
+                $sdk = $route->getLabel('sdk', false);
 
-                if (empty($name)) {
+                if (empty($sdk)) {
                     continue;
                 }
 
-                foreach (Mapper::route($utopia, $route, $complexity) as $field) {
-                    switch ($route->getMethod()) {
-                        case 'GET':
-                            $queries[$name] = $field;
-                            break;
-                        case 'POST':
-                        case 'PUT':
-                        case 'PATCH':
-                        case 'DELETE':
-                            $mutations[$name] = $field;
-                            break;
-                        default:
-                            throw new \Exception("Unsupported method: {$route->getMethod()}");
+                if (!\is_array($sdk)) {
+                    $sdk = [$sdk];
+                }
+
+                foreach ($sdk as $method) {
+                    $namespace = $method->getNamespace();
+                    $methodName = $method->getMethodName();
+                    $name = $namespace . \ucfirst($methodName);
+
+                    foreach (Mapper::route($utopia, $route, $complexity) as $field) {
```

Comment:
We should pass $method into the `route` call, and use $methods->getParameters() to match against the route parameters (instead of iterating route parameters only on line 128). Otherwise both createDocument and createDocuments will have **all** route params in graphql